sugaroid/brain/constants.py

```diff
@@ -162,6 +162,7 @@
 
 I_AM = [
     "lol! I thought I am Sugaroid. have you lost your mind?",
+    "And here I was thinking it was me. Nevermind, I am sure it's just you losing your mind"
```

Comment:
It needs an extra comma at the end of the line I think 
```suggestion
    "And here I was thinking it was me. Nevermind, I am sure it's just you losing your mind",
```